### ISSUE TITLE
PageFieldWidget is missing : bundle.admin.base.min.js

### ISSUE THREAD

**Issue by yml** [divio/django-cms#5595]
There is a regression in 3.3.1 [PageSelectWidget](https://github.com/divio/django-cms/blob/release/3.3.x/cms/forms/widgets.py#L27) is missing the following `Media js`:

* 'cms/js/dist/bundle.admin.base.min.js'

This bug does not exist in 3.3.0.

### How to reproduce the bug ?

* Create a Model with a `PageField`
* register this model in the admin

The JS error : 

```
forms.pageselectwidget.js:19 Uncaught TypeError: $ is not a function(anonymous function) @ forms.pageselectwidget.js:19(anonymous function) @ forms.pageselectwidget.js:83
(index):229 Uncaught TypeError: CMS.$ is not a function(anonymous function) @ (index):229
codeschool_devtools.js:1 '//@ sourceURL' and '//@ sourceMappingURL' are deprecated, please use '//# sourceURL=' and '//# sourceMappingURL=' instead.
``` 

### Work around

Manually add the missing js in Media

```
@admin.register(Location)                                                                                                                                                  
class LocationAdmin(admin.ModelAdmin):                                                                                                                                     
    prepopulated_fields = {"slug": ("name",)}                                                                                                                              
                                                                                                                                                                           
    class Media:                                                                                                                                                           
        js = ('cms/js/dist/bundle.admin.base.min.js',)  
```

### Proper fix

Modify the widget to include `cms/js/dist/bundle.admin.base.min.js` 
There is a good chance that a similar issue exists on all the widgets that are using some custom js that expects `CMS.$`.

**Comment by vxsx** [comment]
Duplicate of https://github.com/divio/django-cms/issues/5476

**Comment by czpython** [comment]
Fixed on 3.3.2
